[bug] pouch pull can't handle space input

### Ⅰ. Issue Description
The following CMD will panic: pouch pull " "

### Ⅱ. Describe what happened
```
#pouch pull " "
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x7944f2]

goroutine 1 [running]:
github.com/alibaba/pouch/pkg/reference.(*taggedReference).Name(0xc420223220, 0xc3dd20, 0xc420223220)
  <autogenerated>:1 +0x32
main.pullMissingImage(0xc40160, 0xc420014098, 0xc45c00, 0xc42009f1d0, 0x7ffd41c9a5ec, 0x1, 0xc42009f101, 0x0, 0x0)
  /tmp/pouchbuild/src/github.com/alibaba/pouch/cli/pull.go:212 +0x146
main.(*PullCommand).runPull(0xc420427d80, 0xc4203eaaa0, 0x1, 0x1, 0x0, 0x0)
  /tmp/pouchbuild/src/github.com/alibaba/pouch/cli/pull.go:60 +0x7d
main.(*PullCommand).Init.func1(0xc420216d80, 0xc4203eaaa0, 0x1, 0x1, 0x0, 0x0)
  /tmp/pouchbuild/src/github.com/alibaba/pouch/cli/pull.go:46 +0x48
github.com/alibaba/pouch/vendor/github.com/spf13/cobra.(*Command).execute(0xc420216d80, 0xc4203eaa60, 0x1, 0x1, 0xc420216d80, 0xc4203eaa60)
  /tmp/pouchbuild/src/github.com/alibaba/pouch/vendor/github.com/spf13/cobra/command.go:742 +0x49b
github.com/alibaba/pouch/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc420216b40, 0xc420293e98, 0x1, 0x1)
  /tmp/pouchbuild/src/github.com/alibaba/pouch/vendor/github.com/spf13/cobra/command.go:827 +0x30e
github.com/alibaba/pouch/vendor/github.com/spf13/cobra.(*Command).Execute(0xc420216b40, 0x1, 0xc4203ea9f0)
  /tmp/pouchbuild/src/github.com/alibaba/pouch/vendor/github.com/spf13/cobra/command.go:780 +0x2b
main.(*Cli).Run(0xc420595400, 0xc3ce20, 0xc420427d70)
  /tmp/pouchbuild/src/github.com/alibaba/pouch/cli/cli.go:95 +0x2f
main.main()
  /tmp/pouchbuild/src/github.com/alibaba/pouch/cli/main.go:53 +0xa9f
```

### Ⅲ. Describe what you expected to happen


### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1.
2.
3.

### Ⅴ. Anything else we need to know?


### Ⅵ. Environment:

- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/cli/pull.go
+++ b/cli/pull.go
@@ -204,7 +204,11 @@ func pullMissingImage(ctx context.Context, apiClient client.CommonAPIClient, ima
 		}
 	}
 
-	namedRef, _ := reference.Parse(image)
+	namedRef, err := reference.Parse(image)
+	if err != nil {
+		return err
+	}
+
 	namedRef = reference.TrimTagForDigest(reference.WithDefaultTagIfMissing(namedRef))
 
 	var name, tag string

--- a/pkg/reference/parse_test.go
+++ b/pkg/reference/parse_test.go
@@ -126,6 +126,11 @@ func TestParse(t *testing.T) {
 				tag:   "1669a6aa7350e1cdd28f972ddad5aceba2912f589f19a090ac",
 			},
 			err: nil,
+		}, {
+			name:     "Contains space input",
+			input:    "  ",
+			expected: nil,
+			err:      ErrInvalid,
 		},
 	} {
 		ref, err := Parse(tc.input)

